flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/serde/FlinkDeserializationContext.java

```diff
@@ -56,6 +56,13 @@ protected FlinkDeserializationContext(
         this.objectMapper = src.objectMapper;
     }
 
+    protected FlinkDeserializationContext(
```

Comment:
How did the version bump affect these lines?

Reply:
The parent class comes from jackson and has an additional abstract method we need to implement.